content/appleboy/drone-sftp-cache/index.md

```diff
@@ -0,0 +1,130 @@
+---
+date: 2017-04-11T00:00:00+00:00
+title: SFTP Cache
+author: appleboy
+tags: [ publish, ssh, sftp, cache ]
+repo: appleboy/drone-sftp-cache
+logo: sftp_cache.svg
+image: appleboy/drone-sftp-cache
+---
+
+Use this plugin for caching build artifacts to speed up your build times. This plugin can create and restore caches of any folders.
+
+```yaml
+pipeline:
+  restore_cache:
+    image: applebot/drone-sftp-cache
+    server: ${SFTP_CACHE_SERVER}
```

Comment:
Lets remove all `${...}`. Secrets should be documented separately in the official documentation. If secrets change (which they do in 0.6) we then have to go through and update all the plugin documentation.